responses/__init__.py

```diff
@@ -656,7 +682,15 @@ def upsert(self, method_or_response=None, url=None, body="", *args, **kwargs):
             self.add(method_or_response, url, body, *args, **kwargs)
 
     def add_callback(
-        self, method, url, callback, match_querystring=False, content_type="text/plain"
+        self,
+        method,
+        url,
+        callback,
+        match_querystring=False,
+        content_type="text/plain",
+        stream=False,
```

Comment:
I didn't think callback responses had streaming.

Reply:
I decided not to touch stream in this PR